README.md

```diff
@@ -29,6 +29,10 @@ Active development occurs in this repository, which means some contracts in it m
 - [`v2-liquidity-mining`](./pkg/liquidity-mining): contracts that compose the liquidity mining (veBAL) system.
 - [`v2-governance-scripts`](./pkg/governance-scripts): contracts that execute complex governance actions.
 
+## Pre-requisites
+
+The build & test instructions below should work out of the box with Node 14.x and Yarn 2.
```

Comment:
```suggestion
The build & test instructions below should work out of the box with Node 14.x and Yarn 2. (Please note that it needs Node 14 specifically, and will NOT work with Node 16 or higher.)
```
I would make this even clearer, since often instructions that mention a version mean "this version or higher."